spring-grpc-core/src/main/java/org/springframework/grpc/client/GrpcChannelFactory.java

```diff
@@ -34,4 +37,16 @@ public interface GrpcChannelFactory {
 	 */
 	ManagedChannelBuilder<?> createChannel(String authority);
 
+	/**
+	 * Creates a {@link ManagedChannelBuilder} for the given authority and the provided
+	 * client interceptors.
+	 * @param authority the target authority for the channel
+	 * @param interceptors the non-null list of interceptors to be applied to the channel
+	 * @param mergeWithGlobalInterceptors whether the provided interceptors should be
+	 * blended with the global interceptors.
+	 * @return a channel builder conifgured with the provided values
+	 */
+	ManagedChannelBuilder<?> createChannel(String authority, List<ClientInterceptor> interceptors,
```

Comment:
I decided not doing a builder for just 1 more method. I think we will want to add one if we add 1 more method to this. Wdyt?